src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -46,6 +39,18 @@ export const post = (req: Request, res: Response) => {
   return res.redirect(getNextPage(req.body[BeneficialOwnerTypeKey]));
 };
 
+export const postSubmit = (req: Request, res: Response) => {
```

Comment:
Again, confused by this change... maybe it's not yours? In which case I would do a local rebase and re-push, which will hopefully help GtHub display only your changes.

Reply:
It is mine, what's the matter with it?

Reply:
Clarified that this logic has moved (from being related to the anchor tag to now being on a button).